### DISCUSSION TITLE
Multiple Choice Dialogs

### DISCUSSION THREAD

**Discussion by ttscoff** [ttscoff/bunch-docs#20] [Show and tell]
The b70 update includes a new feature which allows you to create multiple-choice dialogs that can load different snippets or run different commands based on your selection. The syntax is very coder-ish, with arrays and dictionaries (or objects). I'm open to suggestions on the syntax before it gets nailed down.

Check out [the docs](http://brettterpstra.com//bunch-beta/docs/bunch-files/interactivity/#multiplechoice) for details.

Let me know how it works for you. Maybe this is a feature nobody needs, but I already [modified my Coding.bunch](http://brettterpstra.com/bunch-beta/docs/bunch-files/samplebunch/#coding) to consolidate 3 different Bunches into one, which I think is kinda neato.

**Comment by geofftaylor** [comment]
Nice feature! I like it. I was experimenting with the query dictionary and noticed an inconsistency in the documentation. The doc says the syntax is `key => value`, but the example shows `key : value`. (I'm partial to `key : value`, but I'll use it either way.)

**Comment by ttscoff** [comment]
Oh, right, I changed that after writing the docs. I like the colon 
separator, too, but I got concerned about dealing with cases where there 
was a colon in the key the user wanted, and didn't want to get into 
making the keys/values require quoting. So the old-school ruby-esque 
`=>` seemed like a safe bet. Definitely still willing to reconsider, 
though.

**Reply by geofftaylor** [reply]
Oh yeah, good point. `=>` is better in that case.
